docs/tutorials/create-pages.md

```diff
@@ -0,0 +1,178 @@
+<h1>Tutorial: Creating new pages</h1>
+
+This tutorial will describe how to create new pages and add them to your main menu. It will also cover some steps needed for multilingual implementations.
+
+## Topics covered
+
+* Jekyll pages
+* Markdown
+* Multilingual pages
+* Menu configuration
+
+## Level of difficulty
+
+This tutorial is mostly about content, but does involve the editing of some configuration files. Anyone can follow the steps to complete the tutorial, however you will be using the YAML syntax involved in most Open SDG configurations. YAML requires particular indentation and spacing, so some attention to detail will be needed.
+
+## Decide on a new page to create
+
+In practice, you will need to come up with the title and content for the new page. This may involve some content work, but for now we will just use example text.
+
+## Start a branch
+
+Like many tutorials, we will be changing multiple files. So it is best to create a branch first.
+
+1. In your site repository on Github.com, open the dropdown in the top left which says "develop".
+2. Type "create-pages-tutorial"
+3. In the dropdown select "Create branch: create-pages-tutorial from 'develop'".
+
+This creates a branch called "create-pages-tutorial" and switches to that branch. We will be working on this branch for the rest of the tutorial.
+
+## Start the new page
+
+1. In your site repository on Github.com, go into the `_pages` folder.
+2. Press "Add file" and then "Create new file".
+3. Name your file "example.md". Or substitute any other name, as long as it ends with ".md". This extension stands for [Markdown](https://daringfireball.net/projects/markdown/), which will be discussed in more detail below.
+
+## Add the page's "front matter"
```

Comment:
May want to either define “front matter” immediately or avoid using the term initially.